### PR TITLE
microdata added about twitter account and few more additional details

### PR DESCRIPTION
**Pull request by Nayananga**
I added a twitter account link and a little description about the camel from google search. please let me know if there any correction needed.

### CODE REVIEW

File: antora-ui-camel/src/partials/footer-scripts.hbs [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
-<script type="application/ld+json">
-    {
-    "@context": "http://schema.org/",
-    "@type": "Organization",
-    "url": "{{uiRootPath}}",
-    "logo": "{{uiRootPath}}/img/logo-d.svg"
-    }
+<script type='application/ld+json'>
+{
+  "@context": "http://www.schema.org",
```
Comment (zregvart) [inline antora-ui-camel/src/partials/footer-scripts.hbs]:
```suggestion
  "@context": "http://schema.org",
```

Looking [at the schema.org](https://schema.org/Organization) I think this needs to be without `www`

File: layouts/partials/footer.html [hunk 2]
```diff
@@ -47,13 +47,18 @@ <h5>About</h5>
...
     <script src="{{ "/_/js/site.js" | relURL }}"></script>
     <script src="{{ "/_/js/vendor/highlight.js" | relURL }}"></script>
-    <script type="application/ld+json">
+    <script type='application/ld+json'>
         {
-        "@context": "http://schema.org/",
-        "@type": "Organization",
-        "url": "{{ .Site.BaseURL }}",
-        "logo": "{{ .Site.BaseURL }}_/img/logo-d.svg"
+          "@context": "http://www.schema.org",
```
Comment (zregvart) [inline layouts/partials/footer.html]:
```suggestion
          "@context": "http://schema.org",
```

**Comment by Nayananga** [comment]
Hello Mr.Zoran, Can you check now?

**Comment by zregvart** [comment]
Thank you!

**Pull request merged**